lib/src/timing.dart

```diff
@@ -2,12 +2,19 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+library timing;
```

Comment:
I don't _think_ you need this - the part file should reference this library by uri instead of name now.